app.rb

```diff
@@ -1,20 +1,226 @@
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
 apartments = data[:apartments]
 tenants = data[:tenants]
+ruby_apartments = []
+ruby_tenants = []
+
+puts apartments
+# loop over apartments
 
-# First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
-  # Explain how the data is structured
-  # What are the properties for each of the two types of hashes
+apartments.each do |apartment|
+  Apartment.new
+  ruby_apartments << apartment
```

Comment:
In this line of code it just puts the regular hash that we had before into a different array. Your close, all in all, you need something like this:

```ruby
apartments.each do |apartment|
  ruby_apartment = Apartment.new(id: apartment[:id], address: apartment[:address], rent: apartment[:rent], square_feet: apartment[:square_feet])
  ruby_apartments << ruby_apartment
end
```